### DISCUSSION TITLE
Scan periodically

### DISCUSSION THREAD

**Discussion by allanp35** [aquasecurity/starboard#535] [Questions & Help] [answered]
Hello,

Is there a way to scan periodically images in the cluster using the starboard operator?

Or maybe using a script to activate the operator periodically and scan each pod?

Thank you.

**Comment by danielpacak** [comment, marked as answer]
It's not supported in v0.10 even though we've been talking about such requirement on many occasions.

I realised it was only captured as a TODO note in our issue tracking system so I've transformed the note to issue #537 where we can collaborate and track progress.
